### PR TITLE
Muse Dash: Remove regions for a decent speed gain in generating worlds

### PR DESCRIPTION
**Pull request by DeamonHunter**
## What is this fixing or adding?
Removes the regions for Muse Dash in order to gain a decent amount of performance for larger worlds.

## How was this tested?
Tests and checking the generation log to ensure nothing broke.

Performance testing

image

| Gen | Average Before | Average After |
|--------|--------|--------|
1 x 50 | 0.08304867998 | 0.06629044001 |
10 x 50 | 3.52350716 | 1.65648329  |
1 x 500 | 0.52350967 | 0.31642301  |
10 x 500 | 52.85262655 | 23.17679973  

Tests were not done with set seeds. As the performance difference was consistently large.

### CODE REVIEW

File: worlds/musedash/__init__.py
Comment (nicholassaylor) [inline worlds/musedash/__init__.py:269]:
```suggestion
        # Adds 2 item locations per song/album to the menu region
```
Update the comment to reflect these changes

**Review by nicholassaylor — APPROVED** [review]
Changes LGTM and performance improvements are significant
One suggestion about code comment, but otherwise looks good!

File: worlds/musedash/__init__.py
Comment (DeamonHunter) [reply worlds/musedash/__init__.py]: Should be fixed up. And removed a later one.

**Review by agilbert1412 — APPROVED** [review]
I'm not sure I agree with the idea behind the PR, but it comes from the world maintainer so I defer to their design decisions. I'm actually surprised this improves performance, that goes against what I thought I knew about region performance.
The code seems good, it does what it claims to do

**Review by Exempt-Medic — APPROVED** [review]
Changes LGTM and match the logic of the game. Saw the same speed improvements with my own tests with 10-world and 500-world random generations on the same seeds

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**
